docs/examples/networking/nat/vcn.tf

```diff
@@ -0,0 +1,215 @@
+variable "tenancy_ocid" {}
```

Comment:
This file should probably be called nat.tf - seems to be the pattern we're using.